abi/abi.go

```diff
@@ -16,22 +16,31 @@
 
 package abi
 
-import "github.com/DE-labtory/koa/crpyto"
+import (
+	"encoding/json"
+	"strings"
+)
 
-// The implementation below is implemented for the abi spec of ethereum.
-// https://solidity.readthedocs.io/en/develop/abi-spec.html
+type ABI struct {
+	Methods []Method
+}
 
-// Encode abi parameters
-func Encode(params ...[]byte) []byte {
-	return nil
+func New(abiJSON string) (ABI, error) {
```

Comment:
Why don't you return *ABI?

Reply:
ABI is just value and there is no change in ABI!